src/backend/latex/mod.rs

```diff
@@ -35,4 +40,47 @@ use self::complex::{
     GraphvizGen,
 };
 
+fn inline_figure_begin(out: impl Write, label: &Option<Cow<'_, str>>, caption: &Option<Cow<'_, str>>) -> Result<()> {
```

Comment:
Make this a note in improvements for TODO somewhere, for code clarity and to enhance the feature set of figure. Then I'm content with the situation right now.

Reply:
I did implement your suggestion in a slightly different way: https://github.com/oberien/pundoc/pull/22/files#diff-3eae75b8fbad57dfe22fc3bbf684fa36R43

Reply:
I see, I'd have used an enum with `fn env(&self) -> &'static str` instead of stringly typed `figure`/`table`, in case we want to add more? I didn't consciously use the `float` package yet ^^

Reply:
I encapsulated it by making that string private and only having it in the creation method name. It's more straightforward and provides pretty much the same guarantees.